src/Library/demos/GTK Switch/main.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "GTK Switch",
+  "category": "user_interface",
+  "description": "Overview of GTK Switch",
```

This should be about what `Switch` is, rather than what the entry is about. Remember that Workbench user might not know what "Switch" is.

```suggestion
  "description": "A simple on/off control",
```